packages/react/src/components/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Alert } from './Alert';
+
+describe('Alert', () => {
```

Comment:
Maybe we should consider having a test for testing that the children prop works?

  test('should render children', (): void => {
    render(<Alert severity='warning'><Heading level="1">Alert me!</Heading/></Alert>);
    expect(screen.getByRole('Heading', { level: 1, name: 'Alert me!'}));
  });

Reply:
hehe, it can be simplified by passing just a string and using the getByText method. 😅 Sure, we can also have a test for the className being applied correctly, which I think makes sense to test on this kind of component. But the className itself does not ensure that the styling is working or looks as expected. This component is a great example of a use case for UI/Snapshot testing. But since we are testing our components manually with Storybook I don't think this should be a problem.

Reply:
Added a simple test for `style` and `className` just to check if is applied, may be a bit redundant as we can't actually test if the styling is applied. Actual ui testing would be nice. We have talked about Chromatic or maybe Playwright has some ui testing to? 🤔 

I think most of the ui testing options cost money 🙈

Reply:
I can't see that it breaks any principles of unit testing. We are just isolating the code, leaving styling out. We know that we need `className` in order for styling to work, so it makes sense to test that it's applied.